src/components/ImageView/Toolbar/ToolbarComponent.tsx

```diff
@@ -190,18 +192,20 @@ export class ToolbarComponent extends React.Component<ToolbarComponentProps> {
             </Menu>
         );
 
-        const catalogOverlayEnabled = appStore.activeLayer === ImageViewLayer.Catalog;
         const catalogSelectionDisabled = appStore.catalogNum === 0;
 
         return (
             <ButtonGroup className={className} style={styleProps} vertical={this.props.vertical}>
+                <Tooltip position={tooltipPosition} content={<span>Measure distance<br/><i><small>Click to create geodesic curves.</small></i></span>}>
+                    <AnchorButton icon={<CustomIcon icon="distanceMeasuring"/>} active={appStore.activeLayer === ImageViewLayer.DistanceMeasuring} onClick={() => this.handleActiveLayerClicked(ImageViewLayer.DistanceMeasuring)}/>
+                </Tooltip>
                 <Tooltip position={tooltipPosition} content={<span>Catalog selection<br/><i><small>Click to select single catalog source</small></i></span>}>
-                    <AnchorButton icon={"locate"} active={catalogOverlayEnabled} onClick={() => this.handleActiveLayerClicked(ImageViewLayer.Catalog)} disabled={catalogSelectionDisabled}/>
+                    <AnchorButton icon={"locate"} active={appStore.activeLayer === ImageViewLayer.Catalog} onClick={() => this.handleActiveLayerClicked(ImageViewLayer.Catalog)} disabled={catalogSelectionDisabled}/>
                 </Tooltip>
                 {frame.regionSet.mode === RegionMode.CREATING &&
                 <Tooltip position={tooltipPosition} content={<span>Create region<br/><i><small>Click to select region type</small></i></span>}>
                     <Popover content={regionMenu} position={Position.TOP} minimal={true}>
-                        <AnchorButton icon={regionIcon} active={true}/>
+                        <AnchorButton icon={regionIcon} active={appStore.activeLayer === ImageViewLayer.RegionCreating} onClick={() => this.handleActiveLayerClicked(ImageViewLayer.RegionCreating)}/>
```

Comment:
@TienHao I resolved conflicted with PR https://github.com/CARTAvis/carta-frontend/pull/1441. Please check if this part is ok, thanks.